contracts/modules/uniswap/v2/TernaryLib.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.5.0;
+
+/// @title Library for replacing ternary operator with efficient bitwise operations
+library TernaryLib {
```

Comment:
slick ternary ops in here!

Reply:
One function is a ternary, while the other is not. I wonder if `SortLib` would be more readable and make more sense?

Reply:
Only `sortTokens` is used for sorting. But the rest of the library is meant for replacement of the built-in ternary operator.